### PR TITLE
improve FPS handling + implement cazan popups

### PR DESCRIPTION
**Pull request by Welpike**
- improve FPS handling: adaptation for screens with a lower refresh rate than the fps number required by the game dev
- implement cazan popups: you can use it for show msgs, ask questions (boolean or string user response). You can customize their styling
- updating docs (readme and `./tests/app.js`)
- add basic I/O logic (with popups)
- bundling all declaration files on the end of the build command + implement `yarn run release` that permits to build the project with one min js file, one bundled js file and one .d.ts file
- code cleaning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
  
 *A JS lib that helps you to build games in webpages.*
 
-## Installation
+## Installation 
+
+Nota: Another system for installing is in conception.
```
Comment (Sellig6792) [inline README.md:7]: T'as mis "nota"

File: README.md [hunk 2]
```diff
@@ -33,10 +37,14 @@ window.addEventListener("load", runApp);
...
+    let game = cazan.setup("#yourCanvasId", "2d")
+    game.setSize(600, 350)
+    
+    let rectangle = new cazan.shapes.Shape(game, {x: 10, y: 10}, {x: 50, y: 50})
+
+    game.update()
 }
 ```
-There is a little demonstration of what's possible to do now in `./tests/app.js`!
+There is a little demonstration of what's possible to do actually in `./tests/app.js`!
```
Comment (Sellig6792) [inline README.md:48]: Tu veux dire "for the moment" parce que "actually" c'est "en fait"
```suggestion
There is a little demonstration of what's possible to do for the moment in `./tests/app.js`!
```

File: README.md (hunk 1 above)
Comment (Welpike) [reply README.md:7]: Oui ca se dit (n.b = nota bene), c'est comme confere ou etc

**Review by Sellig6792 — COMMENTED** [review]
Donc quelques détails et pour les FPS on regardera plus tard

**Review by Sellig6792 — APPROVED** [review]

**Pull request merged**